.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Feature Request
+description: Suggest an idea to enhance the current functionality of PyCytominer.
+title: "FeatRequest: "
+labels: [enhancement]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Feature type
+      description: Please check the feature type you would like to propose.
```

Comment:
```suggestion
      description: Please select the feature you would like to propose.
```

Reply:
👍 I added select. I will keep `type of feature` in this description.